screens/overviewMap.js

```diff
@@ -96,6 +96,18 @@ class OverviewMap extends PureComponent {
     };
   }
 
+  // Used to check if we just came from a failed item being pressed
+  componentDidUpdate() {
```

Comment:
I really don't know about this one. It can be dicey with the derived state.

Reply:
Would it be better to do this with a navigation event listener?